core/modules/memman/MemFileSet.h

```diff
@@ -0,0 +1,120 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2016 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_QSERV_MEMMAN_MEMFILESET_H
+#define LSST_QSERV_MEMMAN_MEMFILESET_H
+
+// System headers
+#include <string>
+#include <unistd.h>
```

Comment:
Ah, if only where that simple. unistd.h defines it as an unsigned long while cstddef defines it as unsigned int from what I can see. We definitely want the long version. I suppose I could change that to be uint64_t but I get that value from stat() where it is defined size_t and figured I'd standardize on that definition. I don't have feelings one way or another as long as it 64 bits on 64 bit machines,